tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ["./src/**/*.{svelte,ts,html,js}"],
+  theme: {
+    extend: {},
```

Comment:
Maybe add some tailwind themes? That can be helpful for organizing the development process and keep the colors consistent.

Reply:
I don't think this is needed since it's just a small website with limited pages.

and its not gonna change too much at this current time, right?